app/controllers/house_controller.rb

```diff
@@ -1,11 +1,9 @@
 class HouseController < ApplicationController
-
+end
   def index
-    @houses = Houses.all
+    @houses = House.all
   end
 
   def show
-    @houses = Houses.all
+    @houses = House.all
```

Comment:
The purpose of `show` is to display information about a single house. But `House.all` returns all the houses in the database...